.circleci/config.yml

```diff
@@ -0,0 +1,210 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.8.0
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+
+      - run:
+          name: Install Dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip3 install --upgrade pip
+            pip3 install -r requirements.txt
+
+      - save_cache:
+          key:
+            v1-dependency-cache-{{ checksum "setup.py" }}-{{ checksum "Makefile"
+            }}
+          paths:
+            - 'venv'
+
+      - run:
+          name: Install Resolver
+          command: |
+            . venv/bin/activate
+            pip3 install -e .
+
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - project
+
+  lint-and-unit-test:
+    docker:
+      - image: cloudreach/sceptre-circleci:0.7.0
```

Comment:
Why is this a different version than the one used at the top?

Reply:
fixed